packages/ui-stencil/src/components/internal/orama-embed/orama-embed.tsx

```diff
@@ -0,0 +1,24 @@
+import { Component, h, Prop, State, Listen, Element, Event, type EventEmitter, Watch } from '@stencil/core'
+
+export type ModalStatus = {
+  open: boolean
+  id: HTMLElement
+}
+@Component({
+  tag: 'orama-embed',
+  styleUrl: 'orama-embed.scss',
+  scoped: true,
+})
+export class OramaEmbed {
+  @State() activeElement: HTMLElement
+
+  @Element() el: HTMLElement
+
+  render() {
+    return (
+      <div class="embed">
+        <slot />
+      </div>
```

Comment:
I was thinking... since this layout component here seems to be strictly tight to the searchbox at the moment (this is my assumption looking at the style), I would probably move it right under `orama-search-box` just to make it clear

Reply:
this was my initial implementation, but search-box component started to grow with a lot of code that added useless "noise". by using a dedicated component, it feels more linear and easy to read imo. I also have "the feeling" that embed component will be used as a wrapper for new "embed" components soon.

Reply:
Yes agree, my idea was to move under the folder "orama-search-box" (not the file) until we know we can use it safely in other cases.